utils/misc.go

```diff
@@ -26,6 +26,17 @@ func StringInSlice(str string, sl []string) bool {
 	return false
 }
 
+// IntInSlice returns a bool value if the given int was not/found in the given slice.
```

Comment:
This could be removed in favour of `slices.Contains` https://pkg.go.dev/golang.org/x/exp/slices#Contains

Reply:
I'd love to...   but isn't that still an 'experimental' extension to 1.18?

Reply:
True that, although, it seems like that package will make it out of experimental https://github.com/golang/go/issues/45955

Feel free to ignore my suggestion here.

Reply:
I love your suggestion here!    Would you be comfortable rolling the experimental stuff into production?  If so I'll stick with slices.Contains() (I've already made the changes in my local branch)

Reply:
Yeah sounds good, I'm fine with having the slices experimental package as a dependency, its pretty mature at this point.